### ISSUE TITLE
[Show Page] Responsive About section height with Tabbed Sections new line

### ISSUE THREAD

**Issue by tonyzeoli** [netmix/radio-station#266]
**Describe the bug**

The Posts/Playlists tab gets stuck in a situation where it won't shift all the way left if the Show Description is a certain length. We need to fix this so that if the show description is a certain length, the Posts/Playlists tabs will clear the div and wrap under the left column.

**To Reproduce**
1. Go to Show 14 on RadioStationDemo.com
2. View the Posts/Playlists tabs

**Expected behavior**

Posts/Playlists tabs should wrap under the left column if the Show Description reaches X length.

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by majick777** [comment]
As we discussed, I have matched the Show About Description height to the Show Info sidebar height so that it fills the same vertical area. This has the affect of pushing the tabbed sections below both sections to a new line. If there is no Show Description however, this does not occure, so the tab sections can show in place of that.

Compare:
http://radiostationdemo.com/show/show-11-local-dj-mix/
https://radiostationdemo.com/show/empty-show/

Added to development version 2.3.3.4
